Source/EasyGelf.Log4Net/GelfTcpAppender.cs

```diff
@@ -10,7 +8,7 @@ public sealed class GelfTcpAppender : GelfAppenderBase
     {
         public GelfTcpAppender()
         {
-            TcpTransportConfiguration defaultCfg = TcpTransportConfiguration.GetDefaultConfiguration();
+            var defaultCfg = TcpTransportConfiguration.GetDefaultConfiguration();
```

Comment:
IMHO it's more comfortable to know with which type you are dealing with. So I don't like to use var when right part of the assignment doesn't specify the type explicitly